src/main/java/org/opentripplanner/transit/raptor/api/transit/RaptorTimeTable.java

```diff
@@ -28,4 +28,8 @@
    * Factory method to create the trip search
    */
   RaptorTripScheduleSearch<T> tripSearch(SearchDirection direction);
+
+  default T getHeuristicTrip() {
+    return getTripSchedule(0);
```

Comment:
TOOD: This needs its own implementation which contains the minimum run and dwell times for the pattern.